ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "uuid"
-version = "1.2.1"
+version = "1.2.2"
```

Comment:
This should be the next minor release right?

Reply:
Updated to `1.3.0`